recipes/pyremap/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pyremap" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/xylar/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 3ca8da6fdefaf23f14a4556271e025aef0ab5480a0cae5da1cc3fa1bb46f82f3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - scipy
+    - netCDF4
```

Comment:
Conda package names are always lower case

Reply:
Oops, sorry about that.  I'll need to fix that here and I think in some other places.